### ISSUE TITLE
What should be the basis of record for a preserved specimen which has been discarded?

### ISSUE THREAD

**Issue by ManonGros** [tdwg/dwc-qa#145]
Should the basis of record remain PreservedSpecimen with "discarded" in the disposition term?

**Comment by Jegelewicz** [comment]
That is how specimens in Arctos would appear. I can see how that might be misleading because there is now no specimen, but it does give all the facts that let a person know there was a specimen at one time (and perhaps there are remnants of it somewhere?)

**Comment by tucotuco** [comment]
I agree with this recommendation.

**Comment by Archilegt** [comment]
I agree with keeping "preservedSpecimen" even when the specimen is discarded. This value refers to the source of the original occurrence. 
This is also the situation when doing reverse-digitization. Some collections may want to register in their database legacy literature or catalog records, even if the original specimens do not exist anymore or are unrecognizable because the labels faded or... or... For example, many myriapodological collections are not databased but a few have published catalogs (1862, 1904, etc.). Some myriapodological collections were destroyed by war but their unpublished catalogs survived. In a comprehensive digitization effort, both existing specimens and unmatched printed and written records may be databased as "preservedSpecimen". Reverse-digitization does not change the fact that the recorded occurrence was based on preserved specimens.

**Comment by katiagbif** [comment]
Following this discussion and the correct use of the value preservedSpecimen in the term basisOfRecord, I have a question. What would be the most appropriate term to use when publishing a dataset that refers to specimens preserved in a private collection (even if supported by a publishing institution in GBIF)? This implies that the specimens may not be available for consultation during a visit, wouldn't be available for loans, etc. In some cases, there might not even be a catalog number provided for the specimens in the private collection. We have typically used preservedSpecimen, but could this potentially be confusing for users?

What is your stance on this issue?

Thank you in advance for your insights, and apologies if this topic has already been covered in other threads. :)

**Comment by tucotuco** [comment]
Hi @katiagbif,

I think the [dwc:basisOfRecord](https://dwc.tdwg.org/terms/#dwc:basisOfRecord) is independent of where and how the specimen is maintained or how accessible it is. I would not expect to look at the values of dwc:basisOfRecord to determine if a specimen is in a private collection. There are a couple of things that might help a user understand the situation though. First are the [dwc:institutionCode](https://dwc.tdwg.org/terms/#dwc:institutionCode), [dwc:ownerInstitutionCode](https://dwc.tdwg.org/terms/#dwc:ownerInstitutionCode) and [dwc:collectionCode](https://dwc.tdwg.org/terms/#dwc:collectionCode). These can be spelled out if there are no formal codes for these entities. The second would be to use the [dwc:disposition](https://dwc.tdwg.org/terms/#dwc:disposition) term to capture the private nature of access to the specimen - something like "private collection: inaccessible" or "private collection: accessible". Now that we have a [dwc:MaterialEntity](https://dwc.tdwg.org/terms/#materialentity) class one could also use [dwc:materialEntityRemarks](https://dwc.tdwg.org/terms/#dwc:materialEntityRemarks) to explain exactly what the situation is.

**Comment by katiagbif** [comment]
Thank you very much, @tucotuco, for your prompt and clarifying response. Using these terms will certainly help resolve any potential uncertainties for users consulting the data.

**Comment by tucotuco** [comment]
Excellent.  Good question.

**Comment by ManonGros** [comment]
@katiagbif, if you can, please register the private collection on GRSciColl (https://registry.gbif.org/collection/create) and use the corresponding collection code in the occurrence. You can share a lot of information about the private collection in GRSciColl.

**Comment by katiagbif** [comment]
@ManonGros  Thank you very much for your contribution and suggestion! I am not very familiar with the management of private collections in GrSciColl, so I appreciate the opportunity to ask you about it. When creating a new private collection, is there any recommended practice for setting up the profile of the corresponding institution, or is it better not to assign it to any institution? I have seen various examples and would value any recommendations you can offer to consider for future private collections. Thank you very much!

**Comment by ManonGros** [comment]
@katiagbif A private collection can be registered without any institution (if there is no institutional affiliation and the individual is the sole manager of the collection, we should be able to reflect that). I would suggest to only put the institution if the institution is actually involved in the management of the collection.

**Comment by katiagbif** [comment]
Thank you @ManonGros!
